arkane/reference.py

```diff
@@ -245,5 +250,62 @@ def thermo_data(self, value):
                 raise ValueError('thermo_data for a CalculatedDataEntry object must be an rmgpy ThermoData object')
 
 
+class ReferenceDatabase(object):
+    """
+    A class for loading and working with database of reference species, located at RMG-database/input/reference_sets/
+    """
+    def __init__(self):
+        """
+        Attributes:
+            self.reference_sets (dict): {'set name': [ReferenceSpecies, ...], ...}
+        """
+        self.reference_sets = {}
+
+    def load(self, paths=''):
```

Comment:
Why is `''` the default instead of `None`?

Reply:
No reason other than I set the type hint for this variable a string or list. I'll make this change though